### PR TITLE
Followup changes from the review to the documentation

### PR DESCRIPTION
**Pull request by joecks**

### CODE REVIEW

File: docs/source/getting_started/getting_started.md [hunk 1]
```diff
@@ -216,11 +217,98 @@ Alternatively a history of interactions can be used instead of a user id to ask
...
+## Indexing a filter property
+
+First lets check which properties are already indexed: 
+
+```bash
+curl --location 'https://<url>/documents/_indexed_properties' \
+--header 'authorizationToken: <back_office_token>>'
+```
+
+This returns just the `publication_date`, which is indexed by default (__Deprecated__).
```
Comment (janpetschexain) [inline docs/source/getting_started/getting_started.md:246]: :art: the indexing itself is not deprecated, but the usage of the `published_after` request option is deprecated. the user can still filter by that via the `filter` request option, for example like
```json
"filter": { "publication_date": { "$gte": "2000-01-01T00:00:00Z" } }
```
i'm not sure if we are also going to deprecate the default indexing of `publication_date` in the future, maybe if we see that the tenants start to use their own name for indexing date fields.

File: docs/source/getting_started/getting_started.md [hunk 2]
```diff
@@ -216,11 +217,98 @@ Alternatively a history of interactions can be used instead of a user id to ask
...
+    }
+}'
+
+```
+
+After a short indexing period we can apply filters to our requests.
+
+## Applying a Filter
+
+Applying a filter then just requires to use the `filter` property in the `/semantic_search` or `/users/{user_id}/personalized_documents` query parameter. In the following two examples we simply filter for the tag `conference`.
```
[the remark this answers is not in the document]
Comment (joecks) [reply docs/source/getting_started/getting_started.md:278]: oh really, did it mage it to the version that was applied on 230720150008-a061d03 ?

[the remark this answers is not in the document]
Comment (joecks) [reply docs/source/getting_started/getting_started.md:278]: I checked and yes it is

**Review by janpetschexain — APPROVED** [review]
👍

**Pull request merged**